tests/org/tests_get_group.py

```diff
@@ -0,0 +1,48 @@
+from org.models import *
+from tests.AuthAPITestCase import AuthAPITestCase
+from rest_framework.test import APITestCase, APIClient
+from rest_framework import status
+from users.models import User
+from org.serializers import CreateOrgSerializer
+from org.custom_model_field import PermissionField as Permissions
+import uuid
+
+
+class EditOrgAPITestCase(AuthAPITestCase):
+    def setUp(self):
+        # Inheriting the base class funtionality
+        super(EditOrgAPITestCase, self).setUp()
+
+        data_org = {
+            "name": 'test',
+            "tagline": 'test',
+        }
+        serializer = CreateOrgSerializer(data=data_org)
+        if serializer.is_valid():
+            self.org, self.admin_group = serializer.save()
+
+    def test_fail_without_auth(self):
+        get_org_api = "/api/org/1/"
+        un_auth_client = APIClient()
+        response = un_auth_client.get(get_org_api)
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_fail_with_unknown_org(self):
+        get_org_api = "/api/org/2/"
+        auth_client = self.create_auth_client()
+        response = auth_client.get(get_org_api)
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_pass_with_auth_user(self):
+        get_org_api = "/api/org/1/"
+        auth_client = self.create_auth_client()
+        Member.objects.create(
+            user=self.auth_user,
+            org=self.org,
+            group=self.admin_group
+        )
+        response = auth_client.get(get_org_api)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def tearDown(self):
+        self.auth_user.delete()
```

Comment:
```suggestion
```
1. AuthAPITestCase class automatically destroys this auth_user obj in its tearDown.
1. Since u didn't call the super class tearDown + u have overrided tearDown so self.un_auth_user wont be destroyed at all